### PR TITLE
KYC flow

### PR DESCRIPTION
**Pull request by lizonp**
TODO

- [x] Finish company 'beneficial owner component'

### CODE REVIEW

File: app/components/kyc/shared/KycFileUploadList.tsx [hunk 1]
```diff
@@ -1,32 +1,88 @@
...
+          <li>Full name</li>
+          <li>Date of birth</li>
+          <li>Valid expiration date</li>
+          <li>Official document number</li>
+        </ul>
+      </div>
+      <div className={styles.uploadZone}>
+        <h3 className={styles.title}>Upload documents</h3>
+        {files && <UploadedFiles onRemove={() => {}} files={files} />}
+        <div className={files ? styles.dropzoneWithFilesWrapper : styles.dropzoneWrapper}>
```
Comment (lizonp) [inline app/components/kyc/shared/KycFileUploadList.tsx]: @huesforalice Here is the thing I have trouble with. I need to control layout. Maybe I should do this diferently?

File: app/components/kyc/shared/KycFileUploadList.tsx [hunk 2]
```diff
@@ -1,36 +1,88 @@
...
-    </div>
-  ));
+  const dropzoneWithFilesInner = (
+    <>
+      <InlineIcon svgIcon={plusIcon} />
+      <span>Add more</span>
+    </>
+  );
+  const dropzoneStyle = {
+    width: "210px",
```
Comment (huesforalice) [inline app/components/kyc/shared/KycFileUploadList.tsx]: Please avoid inline styles, and move this into css files. We will have very strict security settings on the production rollout of this, which will disallow inline styles.

**Comment by krzkaczor** [comment]
image

**Comment by krzkaczor** [comment]
and whole form doesnt work :(
image

**Pull request merged**